lib/DataIntegrityProof.js

```diff
@@ -292,6 +292,17 @@ export class DataIntegrityProof extends LinkedDataProof {
       if(!verified) {
         throw new Error('Invalid signature.');
       }
+      if(proof.created) {
+        if(!util.isW3cDate(proof.created)) {
+          throw new Error('invalid datetime format');
+        }
+        const now = new Date();
```

Comment:
We definitely don't want to do this, instead we want to use `this.date`, allowing it to be passed in / set up by the caller to whatever value they want. We might need `date` to default to `null` here (or if it's `undefined` not run the check) to avoid a breaking change. We will need to coordinate this with #32.

Reply:
this turns now into a parameter to `verifyProof` https://github.com/digitalbazaar/data-integrity/pull/34/commits/7ae398140e89a20df6a6b3f830d71b4028011146